packages/aragon-wrapper/src/templates/index.js

```diff
@@ -76,16 +50,13 @@ const Templates = (web3, apm, from) => {
 
       if (!tmplObj) throw new Error('No template found for that name')
 
-      const contractAddress = await apm.getLatestVersionContract(tmplObj.appId)
+      const { contractAddress, abi } = await apm.getLatestVersion(tmplObj.appId)
 
       if (!contractAddress) throw new Error('No template contract found for that appId')
 
-      const template = new web3.eth.Contract(
-        tmplObj.abi,
-        contractAddress
-      )
+      const template = new web3.eth.Contract(abi, contractAddress)
 
-      const token = await newToken(template, tokenParams)
+      const token = await newToken(template, [tokenParams, tokenParams])
```

Comment:
Hack until the client properly sends the token name and token symbol rather than just the organization name as `tokenParams`

Reply:
I think we can remove this now?